src/routes/expenses.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { controllerPostExpense,
+  controllerGetExpenses,
+  controllerGetExpenseById,
+  controllerDeleteExpense,
+  controllerPatchExpense } = require('../controllers/expenses');
```

Comment:
```suggestion
const {
  controllerPostExpense,
  controllerGetExpenses,
  controllerGetExpenseById,
  controllerDeleteExpense,
  controllerPatchExpense,
} = require('../controllers/expenses');
```

Reply:
Still need to add changes according to this comment.